apps/storybook-react/stories/Table.stories.jsx

```diff
@@ -39,9 +39,7 @@ export const simpleTable = () => (
             <Row>
               <Cell>Darth Vader</Cell>
               <Cell>Sith</Cell>
-              <Cell variant="numeric">
-                <Tooltip title="Darth Vader sucks">59</Tooltip>
-              </Cell>
+              <Cell variant="numeric">59</Cell>
```

Comment:
…but it doesn’t remove the problem… The bug is still «The cell alignment is changed from right aligned to center aligned when wrapping cell content with a Tooltip component.» as reported in #408

Reply:
🤣  Fixed now